scripts/run_validation.py

```diff
@@ -57,6 +57,7 @@ def validate_single_rule(cache, datasets, args: Validation_args, rule: dict = No
         ct_package=args.controlled_terminology_package,
         meddra_path=args.meddra,
         whodrug_path=args.whodrug,
+        define_xml_path=args.define_xml_path,
```

Comment:
Also, does the report get the correct define version if a define_xml_path is specified?

Reply:
Okay I have updated as discussed yesterday. 

If I move the define.xml to a separate folder and point to it with the path command it is now populating correctly for both the excel and the json report.

![Screenshot 2023-07-21 at 10 32 27](https://github.com/cdisc-org/cdisc-rules-engine/assets/43621400/359c2610-1abd-4109-8d72-0a63c5fc89d3)
![Screenshot 2023-07-21 at 10 34 14](https://github.com/cdisc-org/cdisc-rules-engine/assets/43621400/9d748e23-6c4c-41bb-a360-9ac7df3f80f9)